package.json

```diff
@@ -40,7 +40,7 @@
     "@nestjs/platform-fastify": "9.0.11",
     "@types/jest": "28.1.7",
     "@types/js-yaml": "4.0.5",
-    "@types/lodash": "4.14.182",
+    "@types/lodash": "4.14.184",
```

Comment:
why do we need to upgrade this? :thinking:

Reply:
It's already updated on master, it's a fail from my rebase: https://github.com/nestjs/swagger/commit/67c78e4bb1abf2a4c882dd040c4e53ae28534053

Reply:
coud you rebase it again? looks like we got some conflicts to fix

Reply:
Sure, it's done